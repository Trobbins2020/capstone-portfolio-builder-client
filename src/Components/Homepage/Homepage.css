 .main_div {
     height: 50vh;
     margin: 5% 20%;
     display: flex;
     flex-direction: column;
     justify-content: space-evenly;
     align-items: center;
     background-color: rgb(42, 176, 230);
     border-radius: 15px;
     color: whitesmoke;
     font-family: 'Arial Narrow Bold', sans-serif;
     box-shadow: -1px -1px 12px 5px rgba(232, 232, 232, 1);
 }

 .getstarted {
     border-radius: 30px;
     border: 2px white solid;
     outline: none;
     padding: 5px 15px;
     background-color: white;
     color: rgb(42, 176, 230);
     box-shadow: -1px -1px 42px 1px white;
 }

 .getstarted:hover {
     background-color: rgb(42, 176, 230);
     color: white;
     box-shadow: none;
 }

 .sub {
     width: 40%;
     word-wrap: break-word;
     letter-spacing: 1px;

 }

 .title {
     font-size: 50px;
     letter-spacing: 1px;
 }

 .sub_buttons {
     width: 40%;
     display: flex;
     justify-content: space-evenly;
     align-items: center;
 }