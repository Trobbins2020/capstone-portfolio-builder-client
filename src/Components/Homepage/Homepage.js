import React, { Component } from "react";
import { Link } from "react-router-dom";
import Header from "../Header/Header";
import "./Homepage.css";
export default class Homepage extends Component {
                 state = {};
                 render() {
                   return (
                     <>
                       <Header />
                       <div className="main_div">
                         <h1 className="text-center title">
                           Time to make your own Portfolio
                         </h1>
                         <h3 className="text-center sub">
                           If you want to make your Portfolio for free, then you
                           came at right place. you can generate your portfolio
                           in few steps
                         </h3>

                         <div className="sub_buttons">
                           <Link
                             to="/get-started"
                             className="getstarted text-center"
                           >
                             Get Started
                           </Link>
                           <Link
                             to="/howtouse"
                             className="getstarted text-center"
                           >
                             How to use it
                           </Link>
                         </div>
                       </div>
                     </>
                   );
                 }
               }
