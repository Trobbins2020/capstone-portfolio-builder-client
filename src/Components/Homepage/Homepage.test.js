import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Homepage from "./Homepage";

describe(`Homepage component`, () => {
  it("renders the Component", () => {
    const wrapper = shallow(<Homepage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
