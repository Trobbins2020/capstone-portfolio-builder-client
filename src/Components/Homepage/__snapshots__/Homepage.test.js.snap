// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage component renders the Component 1`] = `
<Fragment>
  <Header />
  <div
    className="main_div"
  >
    <h1
      className="text-center title"
    >
      Time to make your own Portfolio
    </h1>
    <h3
      className="text-center sub"
    >
      If you want to make your Portfolio for free, then you came at right place. you can generate your portfolio in few steps
    </h3>
    <div
      className="sub_buttons"
    >
      <Link
        className="getstarted text-center"
        replace={false}
        to="/get-started"
      >
        Get Started
      </Link>
      <Link
        className="getstarted text-center"
        replace={false}
        to="/howtouse"
      >
        How to use it
      </Link>
    </div>
  </div>
</Fragment>
`;
